CONTRIBUTING.md

```diff
@@ -1,24 +1,181 @@
-### Contributing to Qt
+## Contributing to Qt
 
-To contribute, fork this project and submit a pull-request.
+Thanks for taking the time to contribute!
 
-Your code will be reviewed and merged once it:
+In here you'll find a series of guidelines for how you can make Qt.py better suit your needs and the needs of the target audience - film, games and tv.
 
-1. Does something useful
-1. Is up to par with surrounding code
+Qt.py was born to address the growing needs in these industries for the development of software capable of running with more than a single flavour of the Qt bindings for Python - PySide, PySide2, PyQt4 and PyQt5.
+
+**Table of contents**
+
+- [Development goals](#development-goals)
+  - [Support co-existence](#support-co-existence)
+  - [Don't get smart](#dont-get-smart)
```

Comment:
It was "simplicity" at first, but I found it too generic. Everyone aims for simplicity.

- "Don't get clever"?
- "Keep it simple"?  (This one is borderline, but better than "simplicity" I guess)